Harubee-iOS/Sources/Presentation/Common/Components/FloatingTitleTextField.swift

```diff
@@ -38,8 +36,12 @@ struct FloatingTitleTextField: View {
         .padding(.leading, 4)
       
       Rectangle()
-        .frame(height: 1)
-        .foregroundStyle(!text.isEmpty ? Color.main : Color.textBrighter)
+        .frame(height: !isTextfieldFocused ? 1 : 2)
+        .foregroundStyle(
+          !isTextfieldFocused
+          ? Color.textBrighter
+          : Color.main
+        )
         .padding(.top, 8)
```

Comment:
포커싱 여부에 따라 Rectangle의 height을 1 또는 2로 지정해주는데, Rectangle의 높이가 변화하면서 그에따라 FloatingTitleTextField의 전체적인 높이도 1씩 변화하고 있습니다.
따라서, Rectangle에 걸려있는 padding 값 역시 포커싱 여부에 따라 값을 달리 줘야할 필요가 있습니다.
(정확한 위치에 고정시키기 위해)
```suggestion
        .frame(height: !isTextfieldFocused ? 1 : 2)
        .foregroundStyle(
          !isTextfieldFocused
          ? Color.textBrighter
          : Color.main
        )
        .padding(.top, !isTextfieldFocused ? 8 : 7)
```